pkgs/development/python-modules/cryptography/default.nix

```diff
@@ -52,6 +54,9 @@ buildPythonPackage rec {
     py.test --disable-pytest-warnings tests
   '';
 
+  # False positive due to both build and host cffi on PYTHONPATH
+  dontUsePythonCatchConflicts = true;
```

Comment:
How should Python path looking when cross-compiling a package that relies on native code, both `host` and `build`? We need to find a solution for this so the hook implementing `dontUsePythonCatchConflicts` functions properly in that case as well.

Reply:
I guess we should make an exception for host and build versions of the same package, but I'm not sure how to implement that. The current hook uses `pkg_resources`, and the `pkg_resources.Distribution` object has a `platform` attribute which seems like it is supposed to do what we want:

> A string representing the platform the distribution is intended for, or None if the distribution is “pure Python” and therefore cross-platform. See Platform Utilities below for more information on platform strings.

This attribute is determined by parsing the name of the `.dist-info` directory, and unfortunately all the packages I have checked don't have a platform string.

Reply:
We probably need two different `PYTHONPATH`s. Until that is solved, we could exclude this hook (see `mk-python-derivation.nix` when cross-compiling.

Reply:
Disabled the check for now in 27d3a18ac7d34b66a14b634cb785e8c8caef024b.